pkg/audit/manager.go

```diff
@@ -1013,6 +1013,9 @@ func logViolation(l logr.Logger,
 	constraint *unstructured.Unstructured,
 	enforcementAction util.EnforcementAction, resourceGroupVersionKind schema.GroupVersionKind, rnamespace, rname, message string, details interface{}, rlabels map[string]string,
 ) {
+	userConstraintAnnotations := constraint.GetAnnotations()
+	delete(userConstraintAnnotations, "kubectl.kubernetes.io/last-applied-configuration")
```

Comment:
i don't think we want this annotation (or maybe any non user set annotations here indeed). lmk if i'm wrong or missing something else